src/delimiter.js

```diff
@@ -339,20 +351,34 @@ const makeStackedDelim = function(
     if (middle === null) {
         // Add that many symbols
         for (let i = 0; i < repeatCount; i++) {
+            inners.push(lap); // overlap
             inners.push(makeInner(repeat, font, mode));
         }
     } else {
         // When there is a middle bit, we need the middle part and two repeated
         // sections
         for (let i = 0; i < repeatCount; i++) {
+            inners.push(lap);
             inners.push(makeInner(repeat, font, mode));
         }
+        // Insert one extra repeat element.
+        inners.push({type: "kern", size: shiftOfExtraElement});
+        inners.push(makeInner(repeat, font, mode));
+        inners.push(lap);
+        // Now insert the middle of the brace.
         inners.push(makeInner(middle, font, mode));
         for (let i = 0; i < repeatCount; i++) {
+            inners.push(lap);
             inners.push(makeInner(repeat, font, mode));
         }
     }
 
+    // To cover the gap create by the overlaps, insert one more repeat element,
+    // at a position that juts 0.005 above the bottom of the top element.
+    inners.push({type: "kern", size: shiftOfExtraElement});
+    inners.push(makeInner(repeat, font, mode));
+    inners.push(lap);
```

Comment:
~In the `else` branch we do the same thing.  Do we need to insert two extra repeat elements?~ I see, there's two `for` loops in the `else` block.